### PR TITLE
Class Territory

### PR DESCRIPTION
**Pull request by symplycharles**

### CODE REVIEW

**Review by maallikm — APPROVED** [review]

File: practical_project/src/main/java/model/Territory.java [hunk 1]
```diff
@@ -1,4 +1,40 @@
 package model;
 
+import javax.persistence.*;
+
+@Entity
+@Table(name = "territory")
```
Comment (kaisicywq) [inline practical_project/src/main/java/model/Territory.java:6]: Table name should be territories

File: practical_project/src/main/java/model/Territory.java [hunk 2]
```diff
@@ -1,4 +1,40 @@
...
 public class Territory {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "territory_id")
+    private int territoryId;
+
+    @Column(name = "descriptionn")
```
Comment (kaisicywq) [inline practical_project/src/main/java/model/Territory.java:14]: Typo on description**n**

File: practical_project/src/main/java/model/Territory.java [hunk 3]
```diff
@@ -1,4 +1,40 @@
...
+    private String name;
```
Comment (kaisicywq) [inline practical_project/src/main/java/model/Territory.java:15]: private String name;   should be  -->  private String description;

File: practical_project/src/main/java/model/Territory.java [hunk 4]
```diff
@@ -1,4 +1,40 @@
...
+
```
Comment (kaisicywq) [inline practical_project/src/main/java/model/Territory.java:16]: missing:

private int regionId;

**Review by kaisicywq — CHANGES_REQUESTED** [review]

**Pull request merged**